src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaFigure {
```

Comment:
create interface `Drawable` with one `void draw()` method
```suggestion
public abstract class Figure implements AreaCalculator, Drawable {
```